### PR TITLE
Add getEntity and getEcryptionKeyIDs functions to key and message types respectively

### PR DESCRIPTION
**Pull request by chesnokovilya**

### CODE REVIEW

File: crypto/key.go [hunk 1]
```diff
@@ -354,6 +354,11 @@ func (key *Key) GetSHA256Fingerprints() (fingerprints []string) {
 	return
 }
 
+// GetEntity get xcrypto Entity object.
```
Comment (wussler) [inline crypto/key.go:357]:
```suggestion
// GetEntity gets x/crypto Entity object.
```

File: crypto/message.go [hunk 2]
```diff
@@ -223,6 +223,24 @@ func (msg *PGPMessage) GetArmoredWithCustomHeaders(comment, version string) (str
 	return armor.ArmorWithTypeAndCustomHeaders(msg.Data, constants.PGPMessageHeader, version, comment)
 }
 
+// getEncryptionKeyIds Returns the key IDs of the keys to which the session key is encrypted.
+func (msg *PGPMessage) getEncryptionKeyIDs() []uint64 {
+	packets := packet.NewReader(bytes.NewReader(msg.Data))
+	var err error
```
Comment (wussler) [inline crypto/message.go:229]: Is it safe to ignore errors here?

File: crypto/message.go [hunk 3]
```diff
@@ -223,6 +223,24 @@ func (msg *PGPMessage) GetArmoredWithCustomHeaders(comment, version string) (str
...
+		var p packet.Packet
+		if p, err = packets.Next(); err == io.EOF {
+			break
+		}
+		enc, ok := p.(*packet.EncryptedKey)
+		if ok {
+			ids = append(ids, enc.KeyId)
+		}
+	}
+	return ids
```
Comment (wussler) [inline crypto/message.go:241]: Shouldn't we issue an error in case IDs is empty?

File: crypto/message.go (hunk 2 above)
Comment (chesnokovilya) [reply crypto/message.go:229]: yes, it is same pattern as in SeparateKeyAndData.

File: crypto/message.go (hunk 3 above)
Comment (chesnokovilya) [reply crypto/message.go:241]: This is matter of preference, for example KeysById in x/crypto could return empty array of keys. In javascript this function could also return empty array. I could add an error, through probably bool flag is better.

File: CHANGELOG.md [hunk 4]
```diff
@@ -16,6 +16,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 (msg *PGPMessage) GetArmoredWithCustomHeaders(comment, version string) (string, error)
 ```
 
+- Now one could get key IDs of keys to which the session key is encrypted
+```go
+func (msg *PGPMessage) getEncryptionKeyIDs() ([]uint64, bool)
+```
+
+get x/crypto Entity (components of an OpenPGP key) from Key struct
```
Comment (wussler) [inline CHANGELOG.md:24]:
```suggestion
- Getter for the x/crypto Entity (internal components of an OpenPGP key) from Key struct
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -16,6 +16,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 (msg *PGPMessage) GetArmoredWithCustomHeaders(comment, version string) (string, error)
 ```
 
+- Now one could get key IDs of keys to which the session key is encrypted
```
Comment (wussler) [inline CHANGELOG.md:19]:
```suggestion
- Extraction of encryption key IDs from a PGP message, i.e. the IDs of the keys used in the encryption of the session key
```

**Pull request merged**